dark/index.html

```diff
@@ -69,13 +69,13 @@
 
     <ul class="nav-links">
       <li>
-        <a href="#home" class="nav-items" id="home" onclick="home()">home</a>

```

Comment:
Don't remove this bcs there is a onclick color given to the every link by that ID. @slayeh17